<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginConfig</class>
 <widget class="QWidget" name="PluginConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="qgbOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="qcbTransmit">
        <property name="toolTip">
         <string>Enable plugins and transmit positional information</string>
        </property>
        <property name="whatsThis">
         <string>This allows plugins for supported games to fetch your in-game position and transmit it with each voice packet. This enables other users to hear your voice in-game from the direction your character is in relation to their own.</string>
        </property>
        <property name="text">
         <string>Link to Game and Transmit Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbPlugins">
     <property name="title">
      <string>Plugins</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="MultiColumnTreeWidget" name="qtwPlugins">
        <property name="accessibleName">
         <string>List of plugins</string>
        </property>
        <property name="accessibleDescription">
         <string>Use up and down keys to navigate through plugins. Use left and right keys to navigate between single plugin permissions.</string>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Enable</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PA</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>KeyEvents</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="qpbReload">
          <property name="toolTip">
           <string>Reloads all plugins</string>
          </property>
          <property name="whatsThis">
           <string>This rescans and reloads plugins. Use this if you just added or changed a plugin to the plugins directory.</string>
          </property>
          <property name="text">
           <string>&amp;Reload plugins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qpbInstallPlugin">
          <property name="toolTip">
           <string>Install a plugin from a local file</string>
          </property>
          <property name="text">
           <string>Install plugin...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="qpbAbout">
          <property name="toolTip">
           <string>Information about plugin</string>
          </property>
          <property name="whatsThis">
           <string>This shows a small information message about the plugin.</string>
          </property>
          <property name="text">
           <string>&amp;About</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qpbConfig">
          <property name="toolTip">
           <string>Show configuration page of plugin</string>
          </property>
          <property name="whatsThis">
           <string>This shows the configuration page of the plugin, if any.</string>
          </property>
          <property name="text">
           <string>&amp;Configure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="qpbUnload">
          <property name="toolTip">
           <string>Unload the currently selected plugin. This will remove it from the plugin list for the current session.</string>
          </property>
          <property name="text">
           <string>Unload</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiColumnTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets/MultiColumnTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
